### PR TITLE
Add instructions for register-project

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

File: rsts/administrator/install/production.rst [hunk 1]
```diff
@@ -154,3 +154,14 @@ Next, open ``propeller/plugins/config.yaml`` and remove the `default-env-vars <h
...
 Run ``kubectl apply -f flyte_generated.yaml`` to deploy these changes to your cluster for a production-ready deployment.
+
+Dynamically Configured Projects
+*******************************
+
+As your Flyte user-base evolves, adding new projects is as simple as registering them through the cli ::
+
+    flyte-cli -h {{ your-flyte-admin-host.com }} register-project --identifier myuniqueworkflow --name FriendlyWorkflowName 
+
+A cron which runs at the cadence specified in flyteadmin config will ensure that all the kubernetes resources necessary for the new project are created and new workflows can successfully
```
Comment (kumare3) [inline rsts/administrator/install/production.rst]: Should we link out to resource controller

Comment (katrogan) [reply rsts/administrator/install/production.rst]: code or docs? cause if the latter it does not yet exist (I want to refactor admin some before writing up the docs - see https://github.com/lyft/flyteidl/pull/24)

Comment (katrogan) [reply rsts/administrator/install/production.rst]: @kumare3

Comment (wild-endeavor) [reply rsts/administrator/install/production.rst]: let's look at this later.

**Pull request merged**